test/linearAlgebra/CSRMatrix.cpp

```diff
@@ -53,15 +53,15 @@ TEST_CASE("CSRMatrix")
         // Sparse
         NeoFOAM::Field<NeoFOAM::scalar> checkSparse(exec, 4);
         auto checkSparseSpan = checkSparse.span();
-        auto csrSparseSpan = sparseMatrixConst.span();
+        auto csrSparseView = sparseMatrixConst.view();
```

Comment:
I think csrView would be sufficient
```suggestion
        auto csrView = sparseMatrixConst.view();
```